libzkbob-rs-wasm/src/client/tx_types.rs

```diff
@@ -129,9 +191,25 @@ impl JsMultiTxType for IMultiTransferData {
             })
             .collect::<Vec<_>>();
 
+            let operator = TxOperator {
+                proxy_address: vec![0; 32], //tx.base_fields.proxy,
```

Comment:
It's an old artifact that is left here from the days when we tried to send multiple transactions at the same time. Since we found that this is not possible in the current solution, this code is no longer used. So I removed it in https://github.com/zkBob/libzkbob-rs/pull/80/commits/0ef282950564d62db21d61c0af96d14402129c75

Reply:
Got it, thank you!